decidim-verifications/README.md

```diff
@@ -59,8 +59,8 @@ Decidim implements two type of authorization methods:
 
   Inside these engines, you can implement any steps required for the
   authorization to succeed, via one or more custom controllers and views. You
-  can create partial `Authorization` records (with the `verified_at` column set
-  to `nil`) and hold partial verification data in the `verification_metadata`
+  can create partial `Authorization` records (with the `verified` column set
```

Comment:
Did you change it, or were the docs wrong? I'd rather have the `verified_at` field than a boolean 😕

Reply:
@mrcasals You're absolutely right! I'm reverting back to the `verified_at` column.

This story is that I started with a datetime column, and then changed to a boolean one because I was running into uniqueness validation problems. However, now that I re-think about it, I don't know why I was insisting on validating the triplet `[:user, :name, :granted(_at)]`, I don't think a user should ever have two authorizations of the same type, so the correct validation is the old one (uniqueness just on `[:user, :name]`)...

Reply:
I've added a few more changes including reverting this change and restoring the `verified_at` column.